### ISSUE TITLE
Default for optional relationship without dependent loading makes no sense

### ISSUE THREAD

**Issue by CasperWSchmidt** [dotnet/EntityFramework.Docs#4502]
Hi

According to the last table on the page, `ClientSetNull` throws an exception, but according to the text right below, `ClientSetNull` is the default. How does it make sense to have a default value that results in exceptions being thrown? I don't know if the table or the text is wrong, but surely the default results in the FK to be set to null somehow 💯 

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 3ed1b8a1-fa32-e147-ae0f-76b3620473f9
* Version Independent ID: 10ce2d9f-8284-a509-5fed-669abfb9d8ae
* Content: [Cascade Delete - EF Core](https://learn.microsoft.com/en-us/ef/core/saving/cascade-delete)
* Content Source: [entity-framework/core/saving/cascade-delete.md](https://github.com/dotnet/EntityFramework.Docs/blob/main/entity-framework/core/saving/cascade-delete.md)
* Product: **entity-framework**
* Technology: **entity-framework-core**
* GitHub Login: @ajcvickers
* Microsoft Alias: **avickers**

**Comment by ajcvickers** [comment]
Agreed the wording could be better here. The last table is specifically what happens when the dependents are _not loaded_. You will get an exception in this case, unless the database is configured for cascades outside of EF Core.

**Comment by CasperWSchmidt** [comment]
I get that the table is specifically for _not loaded_ but as I said, how does it make sense to default this case to `ClientSetNull` when this will throw an error? I guess the default should be `SetNull` no?

Also please clarify how point 1 and 3 is not in conflict :)

**Comment by ajcvickers** [comment]
@CasperWSchmidt `SetNull` would be a wonderful default--but SQL Server support for it is very limited. The point here is that you _must_ load dependents otherwise this scenario will _always_ throw.
